airbyte-integrations/connectors/source-freshdesk/source_freshdesk/spec.json

```diff
@@ -10,22 +10,24 @@
       "domain": {
         "type": "string",
         "description": "Freshdesk domain",
+        "title": "Domain",
         "examples": ["myaccount.freshdesk.com"],
         "pattern": ["^[a-zA-Z0-9._-]*\\.freshdesk\\.com$"]
       },
       "api_key": {
         "type": "string",
+        "title": "API Key",
         "description": "Freshdesk API Key. See the <a href=\"https://docs.airbyte.io/integrations/sources/freshdesk\">docs</a> for more information on how to obtain this key.",
         "airbyte_secret": true
       },
       "requests_per_minute": {
         "title": "Requests per minute",
         "type": "integer",
-        "description": "Number of requests per minute that this source allowed to use."
+        "description": "The number of requests per minute that this source allowed to use."
```

Comment:
can you add an explanation of the default behavior if this option is not set?

Reply:
There is a rate limit of 50 requests per minute per app per account.